src/test/java/bio/terra/service/snapshot/flight/create/SnapshotCreateFlightTest.java

```diff
@@ -157,4 +160,28 @@ void testSnapshotCreateFlightByRequestId() {
             "AddCreatedInfoToSnapshotRequestStep",
             "NotifyUserOfSnapshotCreationStep"));
   }
+
+  @ParameterizedTest
+  @ValueSource(booleans = {true, false})
+  void testSnapshotCreateFlightTDRBillingProfile(boolean tdrBillingProfileFallback) {
+    SnapshotRequestModel request =
+        new SnapshotRequestModel()
+            .addContentsItem(
+                new SnapshotRequestContentsModel()
+                    .mode(SnapshotRequestContentsModel.ModeEnum.BYFULLVIEW));
+    inputParameters.put(JobMapKeys.REQUEST.getKeyName(), request);
+    inputParameters.put(
+        JobMapKeys.TDR_BILLING_PROFILE_FALLBACK.getKeyName(), tdrBillingProfileFallback);
+    var flight = new SnapshotCreateFlight(inputParameters, context);
+
+    if (tdrBillingProfileFallback) {
+      assertThat(
+          FlightTestUtils.getStepNames(flight),
+          containsInRelativeOrder("AuthorizeBillingProfileUseStep"));
```

Comment:
To test a single element in a collection, use `hasItem()`
```suggestion
          hasItem("AuthorizeBillingProfileUseStep"));
```